app/mailers/demo_mailer.rb

```diff
@@ -12,11 +12,12 @@ def invite
   end
 
   def send_worksheets
-    students = params[:students]
+    binding.pry
+    students_email = params[:students]
+    student_group = params[:student_group]
     image_url = params[:image_url]
-    teacher = params[:teacher]
-
-    mail(to: students.map(&:email), subject: 'Hello World')
-    # mail(to: students.map(&:email) << teacher.email, subject: 'Hello World')
+    #teacher_email = params[:teacher]
+    mail(to: students_email, subject: '[#{@student_group.name} for #{@student_group.classroom.name} on Odaiba] Successfully submitted worksheet')
```

Comment:
here you are calling instance variables that haven't been defined in this class

Reply:
but as a heads up: you'll actually need to define instance variables for your view, so you might have to end up defining some instance variables while changing other to regular variables

Reply:
Oh, I see what I did here. So if just remove the '@' in `@student_group.name` it would work. However because I may need it in the body of my view I may as well assign the student_group as an instance variable, right?

Reply:
correct